Lib/tkinter/tix.py

```diff
@@ -63,6 +65,10 @@
 TCL_IDLE_EVENTS   = 1 << 5
 TCL_ALL_EVENTS    = 0
 
+# Deprecation warning
+warnings.warn('the tkinter.tix module is deprecated in favour of tkinter.ttk and is set '
+              'to be removed in the near-future', DeprecationWarning, stacklevel=2)
```

Comment:
"is set to be removed in the near-future" isn't quite accurate.  There are no specific plans to remove it, but it is unmaintained, untested, and will be dropped if it ever fails to build on Windows.  Maybe something like "The Tix Tk extension is unmaintained upstream, and the tkinter.tix wrapper is deprecated in favor of tkinter.ttk."

Though maybe we should actually set a removal target of 3.11 and follow through with it.